src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ActionPlanDTO.kt

```diff
@@ -69,12 +69,14 @@ data class UpdateActionPlanActivityDTO(
 data class ActionPlanSummaryDTO(
   val id: UUID,
   val approvedAt: OffsetDateTime?,
+  val submittedAt: OffsetDateTime?,
 ) {
   companion object {
     fun from(actionPlan: ActionPlan): ActionPlanSummaryDTO {
       return ActionPlanSummaryDTO(
```

Comment:
_not a blocker_
Do we _need_ this endpoint to be summary only? Was this a badly performing call? (People needing this endpoint need all the information anyway)